src/child-applications/child-app-errors.js

```diff
@@ -3,7 +3,7 @@ import CustomEvent from 'custom-event';
 export function handleChildAppError(err, childApp) {
 	const transformedErr = transformErr(err, childApp);
 
-	window.dispatchEvent(new CustomEvent("single-spa:application-broken", {detail: {appName: childApp.name}}));
+	window.dispatchEvent(new CustomEvent("single-spa:application-broken", {detail: {appName: childApp.name, err: transformedErr}}));
```

Comment:
I'm using `evt.detail.err` inside of my tests, and I also think it might be useful to emit the Error object associated with the application being broken, along with this event